docs-ref-conceptual/update-azure-cli.md

```diff
@@ -51,4 +51,15 @@ az config set auto-upgrade.all=no
 If you decide not to use the automatic update feature any more for cases like keeping command scripts running stably, you can turn it off through configuration.
 ```azurecli
 az config set auto-upgrade.enable=no
-```
\ No newline at end of file
+```
+## Update Assistance
+
+Azure cli also provides a command line tool [command-change](https://github.com/Azure/azure-cli-extensions/blob/main/src/command-change/README.md) and a [azure-cli-diff-site](https://azure-cli-diff-site.azurewebsites.net/) to help users justify whether a cli upgrade is safe, how to adjust current command usage by checking the changelist between two cli versions. For detailed usage doc and result analysis, please refer to this [doc](https://github.com/Azure/azure-cli/blob/897a4cd2874e315558d1f1925e866a80ab945f18/doc/breaking_change_detection_guidelines.md).
```

Comment:
**Suggestion 1:** _\[command-change](https://github.com/Azure/azure-cli-extensions/blob/main/src/command-change/README.md)_
We should not be sending customers to GitHub from conceptual content unless it is for a sample.  Can this be changed to a link to our autogenerated reference content?

**Suggestion 2:** Can the `azure-cli-diff-site.azurewebsites.net` be given a H1 name?  The only way for our customers to find this URL is through this one conceptual doc link?

**Suggestion 3:** This information is very informative and I feel needs to be in `azure-docs-cli` published in the same node as our Breaking Changes doc --> _[doc]  \(https://github.com/Azure/azure-cli/blob/897a4cd2874e315558d1f1925e866a80ab945f18/doc/breaking_change_detection_guidelines.md)_

**Suggestion 4:** Can we reword this paragraph to something along the lines of ...

"The Azure CLI also provides a reference command, [az command-change](link to ref content), and a website, [azure-cli-diff-site](https://azure-cli-diff-site.azurewebsites.net/), to help you examine changes between CLI versions.  Use the change list provided to prepare your scripts for a version update.  See \[Breaking change detection for the Azure CLI](link to new doc in azure-docs-cli) for more information.

Reply:
@dbradish-microsoft, thanks for the good suggestions.

- Agree we put the content in https://github.com/Azure/azure-cli/blob/897a4cd2874e315558d1f1925e866a80ab945f18/doc/breaking_change_detection_guidelines.md) to azure-docs-cli and both "Update" and "Upcoming Breaking Changes" nodes can reference that doc.
![image](https://github.com/MicrosoftDocs/azure-docs-cli/assets/16348853/a17b5363-23e9-4560-a56c-2b9fe805561c)
- GitHub repo can have link from learn site as [https://review.learn.microsoft.com/en-us/cli/azure/use-cli-effectively?branch=pr-en-us-3768&tabs=bash%2Cbash2 does](https://learn.microsoft.com/en-us/cli/azure/use-cli-effectively?tabs=bash%2Cbash2) does.
- Regarding SEO, not sure if it's necessary as I don't think customers will get the site via search engine.

-

Reply:
I think this is misleading in the upcoming breaking change since the tools does not show content of future versions.
SEO is important, search is the primary way people discover content nowdays.